### ISSUE TITLE
Bring icon muted colour

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/owncloud-design-system#694]

**Comment by pascalwengerter** [comment]
Perhaps fixed once #1140 is merged into master, which introduces muted colors for all color variations

**Comment by pascalwengerter** [comment]
Closing this since we  now got more color options for icons (and muted isn't necessary currently AFAIK)
